### DISCUSSION TITLE
支持动态服务发现

### DISCUSSION THREAD

**Discussion by akaxb** [bfenetworks/bfe#917] [Q&A (问题和回答)] [answered]
我看到了路由示例，发现示例中是在配置文件中写死的服务实例的IP地址，所以我想请问下，BFE支持动态的服务发现吗，比如集成了eureka、consul，如果支持，是否可以提供相关的文档

**Comment by iyangsj** [comment]
BFE开源版本移除了对内部服务发现的支持，后续版本会集成兼容开源服务发现项目

**Comment by zhangjimmy** [comment]
能支持nacos作为服务发现

**Comment by lemonlinger** [comment]
是否考虑 [xDS](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol) 协议的实现？

**Comment by buchongyu** [comment]
> BFE开源版本移除了对内部服务发现的支持，后续版本会集成兼容开源服务发现项目

咱们bfe有没有支持eureka等动态服务注册的时间点呢？

**Comment by mileszhang2016** [comment, marked as answer]
可以利用BFE API提供的接口来更新实例池中的实例信息，详见
https://github.com/bfenetworks/api-server/blob/develop/docs/zh_cn/open_api/product/product_pools.md
